cmd/ctr/commands/commands_windows.go

```diff
@@ -21,11 +21,20 @@ import (
 )
 
 func init() {
-	ContainerFlags = append(ContainerFlags, cli.Uint64Flag{
-		Name:  "cpu-count",
-		Usage: "number of CPUs available to the container",
-	}, cli.StringSliceFlag{
-		Name:  "device",
-		Usage: "identifier of a device to add to the container  (e.g. class://5B45201D-F2F2-4F3B-85BB-30FF1F953599)",
-	})
+	ContainerFlags = append(ContainerFlags,
+		cli.Uint64Flag{
+			Name:  "cpu-count",
+			Usage: "number of CPUs available to the container",
+		},
+		cli.Uint64Flag{
+			Name:  "cpu-shares",
+			Usage: "The relative number of CPU shares given to the container relative to other workloads. Between 0 and 10,000.",
+		},
+		cli.Uint64Flag{
+			Name:  "cpu-max",
+			Usage: "The number of processor cycles threads in a container can use per 10,000 cycles. Set to a percentage times 100. Between 0 and 10,000",
```

Comment:
I think this actually has to be at least 1. Under the hood this translates to setting via [`JOBOBJECT_CPU_RATE_CONTROL_INFORMATION`](https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-jobobject_cpu_rate_control_information), and the `CpuRate` member can't be 0.

Reply:
Good catch! Thanks Kevin